WTWSMS/events/WtWSMS_migration_events.txt

```diff
@@ -357,12 +357,10 @@ character_event = {
 
 # Slavic Migration
 province_event = {
-	id = zusk_event.8
-	desc = zusk_event.8.desc
-	picture = GFX_evt_tribal_life
+	id = zusk_event.80
```

Comment:
I would advice to use another ID, such as the latest one available in that range.

Reply:
☑️